adafruit_sgp40.py

```diff
@@ -117,19 +155,74 @@ def _reset(self):
         try:
             self._read_word_from_command(delay_ms=50)
         except OSError:
-            # print("\tGot expected OSError from reset")
+            # Got expected OSError from reset
             pass
         sleep(1)
 
+    @staticmethod
+    def _temp_c_to_ticks(temperature):
+        """
+        Converts Temperature in Celsius to 'ticks' which are an input parameter
+        the sgp40 can use
+
+        Temperature to Ticks : From SGP40 Datasheet Table 10
+        temp (C)    | Hex Code (Check Sum/CRC Hex Code)
+            25      | 0x6666   (CRC 0x93)
+            -45     | 0x0000   (CRC 0x81)
+            130     | 0xFFFF   (CRC 0xAC)
+
+        """
+        temp_ticks = int(((temperature + 45) * 65535) / 175) & 0xFFFF
+        least_sig_temp_ticks = temp_ticks & 0xFF
+        most_sig_temp_ticks = (temp_ticks >> 8) & 0xFF
+
+        return [most_sig_temp_ticks, least_sig_temp_ticks]
+
+    @staticmethod
+    def _relative_humidity_to_ticks(humidity):
+        """
+        Converts Relative Humidity in % to 'ticks' which are  an input parameter
+        the sgp40 can use
+
+        Relative Humidity to Ticks : From SGP40 Datasheet Table 10
+        Humidity (%) | Hex Code (Check Sum/CRC Hex Code)
+            50       | 0x8000   (CRC 0xA2)
+            0        | 0x0000   (CRC 0x81)
+            100      | 0xFFFF   (CRC 0xAC)
+
+        """
+        humidity_ticks = int((humidity * 65535) / 100 + 0.5) & 0xFFFF
+        least_sig_rhumidity_ticks = humidity_ticks & 0xFF
+        most_sig_rhumidity_ticks = (humidity_ticks >> 8) & 0xFF
+
+        return [most_sig_rhumidity_ticks, least_sig_rhumidity_ticks]
+
     @property
     def raw(self):
         """The raw gas value"""
         # recycle a single buffer
-        self._command_buffer = bytearray(_READ_CMD)
+        self._command_buffer = self._measure_command
         read_value = self._read_word_from_command(delay_ms=250)
         self._command_buffer = bytearray(2)
         return read_value[0]
 
+    def measure_raw(self, temperature=25, relative_humidity=50):
+        """
```

Comment:
Please document the parameters used here
:param temperature ...
:param humidity  ...